manifests/client/params.pp

```diff
@@ -4,7 +4,7 @@
   $manage_package = pick($mongodb::globals::manage_package, $mongodb::globals::manage_package_repo, false)
 
   if $manage_package {
-    $package_name = "mongodb-${mongodb::globals::edition}-shell"
+    $package_name = pick($mongodb::globals::client_package_name, "mongodb-${mongodb::globals::edition}-shell")
```

Comment:
You should define `$client_package_name` in `mongodb::globals`.

Reply:
It's actually already defined. It just doesn't get called. Probably a miss in the upstream